man/gbz80.7

```diff
@@ -754,22 +754,30 @@ Cycles: 1
 Bytes: 1
 .Pp
 Flags: None affected.
-.Ss JR e8
-Relative Jump by adding
-.Ar e8
-to the address of the instruction following the
-.Sy JR .
-To clarify, an operand of 0 is equivalent to no jumping.
+.Ss JR n16
+Relative Jump to address
+.Ar n16 .
+The address is encoded as a signed 8-bit offset from the end of the
+.Sy JR
```

Comment:
```suggestion
.Cm JR
```

Reply:
What's the difference? I don't see `.Cm` used elsewhere here.

Reply:
Sorry,
```suggestion
.Ic JR
```
is the correct one. (Consistent with rgbasm(5), too.)

`Sy` here is on the same level as using `<b>` when you mean `<strong>`; if it's the case elsewhere in the document, then they'll have to be fixed too.